config_sample.yml

```diff
@@ -10,7 +10,9 @@ captchaPath: c:/selenium-config/captcha.txt #Path to your captcha.txt file
 qs: null
 
 #Config related with mobile devices
-platfrom: null #acceptable values: IOS and ANDROID
+platfrom: ANDROID #acceptable values: IOS and ANDROID
```

Comment:
For now, Appium is only working on Android devices, once it works on iOS as well, we will set it to default